src/Keystore/StoredKey.cpp

```diff
@@ -250,7 +250,7 @@ const PrivateKey StoredKey::privateKey(TWCoinType coin, const Data& password) {
     return privateKey(coin, TWDerivationDefault, password);
 }
 
-const PrivateKey StoredKey::privateKey(TWCoinType coin, TWDerivation derivation, const Data& password) {
+const PrivateKey StoredKey::privateKey(TWCoinType coin, [[maybe_unused]] TWDerivation derivation, const Data& password) {
```

Comment:
@catenocrypt `derivation` isn't used here?

Reply:
Indeed, good catch. Effectively default derivation is used, not the input parameter.
Fix is small, but I prefer to do it in separate PR, not hide in this large changeset.